openshift/templates/drift-backend.bc.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     app: drift-backend
   name: drift-backend
-  namespace: {{ project }}
+  namespace: "{{ project }}"
```

Comment:
I didn't realize quoting the jinja templates allow it to pass yamllint. I'll add this back for the openapi specs after this PR is merged.

Reply:
@ehelms looks like this is getting picked up by your check already:

```
$ yamllint drift/openapi/api.spec.yaml
drift/openapi/api.spec.yaml
  12:12     error    too many spaces inside braces  (braces)
  12:24     error    too many spaces inside braces  (braces)
  12:27     error    syntax error: expected <block end>, but found '{'
  12:29     error    too many spaces inside braces  (braces)
  12:38     error    too many spaces inside braces  (braces)
The command "yamllint drift/openapi/api.spec.yaml" exited with 1.
```

I'm guessing it just needs a handful of `"` thrown in on line 12.

Reply:
I had to update that in the spec for the tests to pass in this PR.